### DISCUSSION TITLE
Sound Issues - No Output or Input Device

### DISCUSSION THREAD

**Discussion by MateusSousa0** [elementary/triage#205] [Q&A]
Could someone please help me?

I'm having problems with the sound. In system settings it says "No output device" and "No input device" and I can't hear anything. I recently switched from Windows to Linux and am using elementary OS 7 Horus.

The driver is: snd_hda_intel
And the audio controller model is: ALC887-VD.

**Comment by jeremypw** [comment]
Seems like this controller has caused trouble on a number of Linux distributions.  See for example https://unix.stackexchange.com/questions/691396/how-to-get-realtek-alc887-vd-to-play-any-sound-or-be-recognized-in-debian-11-fr.  Or search Google for "Linux RealTek ALC887-VD". Elementary is based on Ubuntu which is itself based on Debian so solutions on those platforms may work well work on Elementary too.  Hopefully an Elementary user with the same controller will be able to give a solution.

**Comment by jeremypw** [comment]
There seems to be the outline of a solution here: https://gist.github.com/kiview/7540d2e1e8a11ef9f6e1c09bef388d46

**Comment by 24glinka** [comment]
I tried what suggested but nothing, the audio doesn't work. I have "**Audio device: Intel Corporation Haswell-ULT HD Audio Controller**".
What should I do to reactivate it? 
I have elementary OS 7.1 Horus installed on Mac.
